ui/app/src/utils/itemActions.ts

```diff
@@ -369,15 +369,13 @@ export function generateSingleItemOptions(
   // endregion
 
   // region Section C
-  if (hasPublishAction(item.availableActions) && actionsToInclude.publish) {
+  if (
+    (hasPublishAction(item.availableActions) && actionsToInclude.publish) ||
+    (hasPublishRequestAction(item.availableActions) && actionsToInclude.requestPublish) ||
+    (hasApprovePublishAction(item.availableActions) && actionsToInclude.approvePublish)
+  ) {
     sectionC.push(menuOptions.publish);
   }
-  if (hasPublishRequestAction(item.availableActions) && actionsToInclude.requestPublish) {
-    sectionC.push(menuOptions.requestPublish);
-  }
-  if (hasApprovePublishAction(item.availableActions) && actionsToInclude.approvePublish) {
-    sectionC.push(menuOptions.approvePublish);
-  }
   if (hasSchedulePublishAction(item.availableActions) && actionsToInclude.schedulePublish) {
     sectionC.push(menuOptions.schedulePublish);
   }
```

Comment:
Let's leave the publish option only.
(Unless you see otherwise on the [slack thread](https://craftersoftware.slack.com/archives/C0EMZALLV/p1625852285192800).)